Added validations to models

tinajohnson opened the pull request.

tinajohnson committed:
Added validations to models

sitharapri commented on .idea/app.iml:
```diff
@@ -232,6 +232,26 @@
       </library>
     </orderEntry>
     <orderEntry type="module-library">
+      <library name="babel-source (vbundled(5.8.35), /home/tina/RubymineProjects/app/vendor/bundle/gems/babel-source-5.8.35) [path][gem]">
```
Please remove the entire .idea folder itself.

tinajohnson committed:
Remove .idea directory

tinajohnson committed:
The following changes were made:

1. Renamed active admin models to match convention
2. Renamed column names of tables.
3. Removed teacher model(We will be going with a user model)
4. Remove teachers controllers.
5. Added missing migrations.
6. Removed nnecessary helpers.

sitharapri commented on .idea/workspace.xml:
```diff
@@ -2,11 +2,59 @@
 <project version="4">
   <component name="ChangeListManager">
     <list default="true" id="f8d56295-038f-4317-a891-da55163d8410" name="Default" comment="">
```
Please remove this file

sitharapri commented on app/models/class_mapping.rb:
```diff
@@ -1,8 +1,5 @@
-  belongs_to :standard
-  belongs_to :section
-  belongs_to :school
+  belongs_to :section, class_name: Section, foreign_key: :section_id
```
Why mentioned class_name here?

sitharapri commented on app/views/sections/show.json.jbuilder:
```diff
@@ -1 +1 @@
-json.extract! @section, :id, :section_name, :string, :created_at, :updated_at
+json.extract! @section, :id, :name, :created_at, :updated_at
```
Why do we need these?

tinajohnson committed:
Minor changes
